python/ray/llm/_internal/batch/processor/vllm_engine_proc.py

```diff
@@ -58,6 +58,16 @@ class vLLMEngineProcessorConfig(OfflineProcessorConfig):
         "specified and LoRA is enabled, then the 'model' in LoRA "
         "requests will be interpreted as model ID used by HF transformers.",
     )
+    # Placement group config for TP/PP.
+    placement_group_config: Optional[Dict[str, Any]] = Field(
+        default=None,
+        description=(
+            "Custom Ray placement group configuration for scheduling vLLM engine workers. "
+            "Each bundle should define its resource requirements, such as 'CPU' and 'GPU'. "
+            "Note: When using vLLM's Ray distributed executor backend, each bundle must be restricted "
+            "to a single GPU. This configuration is only applicable when the Ray distributed executor backend is enabled."
+        ),
+    )
```

Comment:
We need to document better what this structure represent and how it should be constructed. Is it only the bundles? Is it gonna be bundles + strategy? Is it a placement group object?

Reply:
I'd create a schema data class for placement_group_config here: 

```
class PlacementGroupSchema:
    bundles: int[dict[str, int]]
    strategy: Literal[PACK, STRICT_PACK, SPREAD, STRICT_SPREAD]
```

and then I'd cast the passed dict to this data type. we can then later do `placementgroup(**pg.dict())` or sth similar to construct the real placement group.